src/Pages/NavigatorDashboard.js

```diff
@@ -1,145 +1,162 @@
-import React, { useEffect, useState } from "react";
-import { collection, query, where, getDocs, doc, update } from "firebase/firestore";
-import { db } from "../Components/firebase";
-import "./NavigatorDashboard.css"
-import Checkboxes from "./Checkboxes";
+import React, { useEffect, useState } from 'react';
```

Comment:
Cleanup of profile page including reformatting, reordering variable declarations and prop type validation